env.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  FALSE_VALUE: false,
```

Comment:
These four are just examples - this file can basically be:

```
module.exports = {};
```

Reply:
Edited